fbw-a32nx/src/systems/fmgc/src/flightplanning/legs/FlightPlanLeg.ts

```diff
@@ -303,17 +303,26 @@ export class FlightPlanLeg implements ReadonlyFlightPlanLeg {
     return this;
   }
 
-  clearConstraints() {
-    this.definition.verticalAngle = undefined;
+  clearAltitudeConstraints() {
     this.definition.altitudeDescriptor = undefined;
```

Comment:
**thought:** 😬 we should really use `.editLegDefinition` to accomplish this... for later I guess. this currently breaks sync